### PR TITLE
Check chart labels on all objects

### PR DESCRIPTION
**Pull request by jedcunningham**
We weren't checking labels on all of the objects our chart creates.
Instead of making it fatal, I've added a warning so new objects that are
forgotten are more visible.

### CODE REVIEW

File: chart/templates/secrets/metadata-connection-secret.yaml [hunk 1]
```diff
@@ -31,6 +31,8 @@ apiVersion: v1
 metadata:
   name: {{ .Release.Name }}-airflow-metadata
   labels:
+    tier: airflow
+    component: connection
```
Comment (jedcunningham) [inline chart/templates/secrets/metadata-connection-secret.yaml]: I'm considering not setting a component for some of these, like this one. Thoughts on having it on "most but not all"?

Comment (ashb) [reply chart/templates/secrets/metadata-connection-secret.yaml]: Yeah, this one feels wrong, especially as it used by all "components"

**Review by ashb — COMMENTED** [review]
I'm not sure that having component on all the jobs/secrets etc really makes sense.

Doubly so when having the component is only used for a single object.

**Comment by jedcunningham** [comment]
@ash, I removed some of them. I did leave them on the jobs, however, as we do have existing component labels on the `Job` objects.

**Review by kaxil — APPROVED** [review]

**Pull request merged**